src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        try {
```

Comment:
try to split this logic into 3 private methods
1. read a string from the file
2. convert the string to string-report
3. write report to file